### DISCUSSION TITLE
Add Delete / Remove icon to each image

### DISCUSSION THREAD

**Discussion by pepsiamir** [igordanchenko/react-photo-album#138] [Q&A] [answered]
Is it possible to add remove image from stack?

I have no idea how to pass the images array as prop from PhotoAlbum to this component so I can remove one of them by id. Can you help please?

```tsx
const PhotoFrame = memo(
  forwardRef<HTMLDivElement, PhotoFrameProps>((props, ref) => {
    const {
      layoutOptions,
      imageProps,
      activeIndex,
      overlay,
      active,
      insertPosition,
      attributes,
      listeners,
      setActivatorNodeRef,
    } = props;
    const { alt, style, ...restImageProps } = imageProps;
```

**Comment by igordanchenko** [comment, marked as answer]
It sounds like you are building on top of the [Sortable Gallery](https://react-photo-album.com/examples/sortable) example. If that's the case, then you can add `onRemove` callback to the `PhotoFrame` component and implement that callback inside the `renderPhoto` render function. 

Here is a working example - https://codesandbox.io/p/sandbox/react-photo-album-138-9n3hh2?file=%2Fsrc%2FApp.tsx

**Reply by pepsiamir** [reply]
Thank you @igordanchenko. It worked
